internal/cli/version.go

```diff
@@ -42,6 +44,7 @@ func DefaultVersionOptions() *VersionOptions {
 	return &VersionOptions{
 		GlobalOptions: DefaultGlobalOptions(),
 		Output:        "",
+		Timeout:       5 * time.Second,
```

Comment:
Should be a named constant rather than a "magic number".

Reply:
Should there be a way to specify "no timeout", e.g. when the timeout is set to "0"?

Reply:
removed